tests/system/Log/LoggerTest.php

```diff
@@ -63,7 +64,7 @@ public function testLogActuallyLogs(): void
         $config = new LoggerConfig();
         $logger = new Logger($config);
 
-        $expected = 'DEBUG - ' . date('Y-m-d') . ' --> Test message';
+        $expected = 'DEBUG - ' . Time::now()->toDateString() . ' --> Test message';
         $logger->log('debug', 'Test message');
```

Comment:
This codes is still time-dependent.
If `Time::now()` is called on today 23:59:59.9999 and ` $logger->log()` is called on tomorrow 0:00:00, it would fail.
The solution is to fix the current time:
https://codeigniter4.github.io/CodeIgniter4/testing/overview.html#testing-and-time

Reply:
But how should we make the `$logger->log()` use the test time we set?

```php
 public function testLogActuallyLogs(): void
    {
        $config = new LoggerConfig();
        $logger = new Logger($config);

        Time::setTestNow('2023-11-25 12:00:00');

        $expected = 'DEBUG - ' . (string) Time::now()->format('Y-m-d') . ' --> Test message';
        $logger->log('debug', 'Test message');

        $logs = TestHandler::getLogs();

        $this->assertCount(1, $logs);
        $this->assertSame($expected, $logs[0]);
    }
```

There seems to be no way to inject test time into the logger.

Reply:
`Time::setTestNow()` is a static method. It has a static property, and if you use `Time::now()`, it returns the test time you set.

Reply:
Done.